CMakeLists.txt

```diff
@@ -232,13 +231,14 @@ set(_acts_podio_version 1.0.1) # will try this first
 set(_acts_podio_fallback_version 0.16) # if not found, will try this one
 set(_acts_doxygen_version 1.9.4)
 set(_acts_hepmc3_version 3.2.1)
-set(_acts_nlohmanjson_version 3.2.0)
```

Comment:
This also ends up being the **minimum** required version. Is this version available everywhere? (I think it probably is)

Reply:
Understood, we also have some `_recommended_verison` for `boost` I guess.